lib/prawn/table/cell.rb

```diff
@@ -191,6 +187,15 @@ def self.make(pdf, content, options={})
         end
       end
 
+      def self.stringify_content?(content)
+        return true if content.nil?
+        return true if content.kind_of?(Numeric)
+        return true if content.kind_of?(Date)
+        return true if content.kind_of?(Time)
```

Comment:
Why do you use separate if statements instead of ||-ing them together?

Reply:
Readability / simpler conditions.  If there's a desire to concatenate I've got no huge objection.

Reply:
It's fine as it is. Just wanted to know the reason.